app/routes/articles.js

```diff
@@ -30,10 +30,53 @@ router.post('/', isLoggedIn, (req, res) => {
 })
 
 //NEW - Show form to create new article
-router.get('/new', isLoggedIn, (req, res) => {
+router.get('/new', isLoggedIn, hasAuthorRole, (req, res) => {
     res.render('articles/new.ejs')
 })
 
+//APPROVE List Article Route
+router.get('/approve', isLoggedIn, (req, res) => {
+    if(!req.user.isAdmin) {
+        req.flash('Oops! Something went wrong!')
```

Comment:
Not authorized instead?

Reply:
I don't want the standard user to be aware of the hidden URL, hence the normalized error message.